2020-01-18  Jakub Jelinek  <jakub@redhat.com>

	* coroutines.cc (get_fn_local_identifier): Fix NO_DOT_IN_LABEL
	but non-NO_DOLLAR_IN_LABEL case build.

--- gcc/cp/coroutines.cc.jj	2020-01-18 13:47:09.318360691 +0100
+++ gcc/cp/coroutines.cc	2020-01-18 19:05:48.349119608 +0100
@@ -2252,7 +2252,7 @@ get_fn_local_identifier (tree orig, cons
   sep = ".";
 #else
 #ifndef NO_DOLLAR_IN_LABEL
-  sep = "$"
+  sep = "$";
 #else
   sep = "_";
   pfx = "__";
